hat/assets/js/apps/Iaso/domains/app/components/SidebarMenuComponent.js

```diff
@@ -141,13 +141,6 @@ const SidebarMenu = ({
             </List>
             <Box className={classes.user}>
                 <LanguageSwitch />
-                <Typography
```

Comment:
I understand we don't really need it there anymore since it's in the top bar, but then when we switch to mobile layout (eg budget) we don't display the username anywhere.

Reply:
Username is displayed on mobile in the sidebar